style/modules/subFooter.css

```diff
@@ -0,0 +1,44 @@
+.subFooter {
+    display: block;
```

Comment:
this line doesn't do anything because it's superseded by the `display: flex` & should be removed

(you can only have one `display` value, so the last one wins -- luckily flex is already "block-like")